### PR TITLE
refactor: use top-level .gitignore & .travis.yml

### PR DESCRIPTION
**Pull request by virkt25**
Follow up based on a review comment here:
https://github.com/strongloop/loopback-next/pull/858#discussion_r161526236

This removes package level .gitignore and .travis.yml files in favour
of top-level files.

## Checklist

- [x] `npm test` passes on your machine
- [n/a] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [n/a] Related API Documentation was updated
- [n/a] Affected artifact templates in `packages/cli` were updated
- [n/a] Affected example projects in `packages/example-*` were updated

### CODE REVIEW

File: packages/cli/.gitignore [hunk 1]
```diff
@@ -1,3 +0,0 @@
-coverage
-node_modules
```
Comment (shimks) [inline packages/cli/.gitignore]: do we need a line for `cli/node_modules` in the top-level as well?

Comment (raymondfeng) [reply packages/cli/.gitignore]: I don't think so. `node_modules` in top level `.gitignore` should have it covered.

**Comment by rmg** [comment]
@slnode test please

File: packages/cli/.gitignore (hunk 1 above)
Comment (virkt25) [reply packages/cli/.gitignore]: top level has a `node_modules` folder which works for all sub-directories. `sandbox` was specific to cli so it was marked as such.

Comment (virkt25) [reply packages/cli/.gitignore]: oops didn't see your response there ... but what @raymondfeng said basically.

Comment (shimks) [reply packages/cli/.gitignore]: 👍

**Pull request merged**